web-client/public/js/grnstate.js

```diff
@@ -71,6 +71,10 @@ export const grnState = {
     dashedLine: false,
 
     annotateLinks: () => annotateLinks(currentNetwork),
+
+// Zoom Parameter
+    zoomValue: 4,
```

Comment:
@mihirsamdarshi Actually the protocol is you _do_ wait for an approving review before merging a PR even after you have responded to feedback. Because, for example, you might miss something like this. The earlier `4` was replaced with `DEFAULT_ZOOM_VALUE`, but what about this one?

The code reviewer is supposed to explicitly re-approve a PR before it gets merged, precisely because there might be loose ends like the one above.